python/tvm/tir/analysis/analysis.py

```diff
@@ -331,3 +331,32 @@ def OOBChecker():
         The result pass
     """
     return _ffi_api.OOBChecker()  # type: ignore
+
+
+def communicative_deep_equal(lhs: PrimExpr, rhs: PrimExpr) -> bool:
```

Comment:
Is "communicative" a word? Do you mean "commutative"?

Reply:
Thanks for the review！It was a typo, I have corrected it.